src/toil/test/jobStores/jobStoreTest.py

```diff
@@ -1458,27 +1456,6 @@ def testInvalidJobStoreName(self):
                           AzureJobStore,
                           'toiltest:a_b')
 
-@needs_aws
-@needs_encryption
-@slow
-class EncryptedAWSJobStoreTest(AWSJobStoreTest, AbstractEncryptedJobStoreTest.Test):
-    pass
-
-
-@needs_azure
-@needs_encryption
-@slow
-class EncryptedAzureJobStoreTest(AzureJobStoreTest, AbstractEncryptedJobStoreTest.Test):
-    pass
```

Comment:
Are you deleting these just because the class body is empty? I think the EncryptedAzureJobStoreTest and the EncryptedAWSJobStoreTest classes really do create actual tests that can be run, because they inherit other classes and compose them together. Basically each of these says "run this set of tests using this job store implementation".

I suspect that we want to keep these (or at least the AWS one) to be part of the integration tests that @DailyDreaming is getting set up again.

Reply:
That's my bad. I didn't realize it was running the AbstractEncryptedJobStoreTest. I will revert this change.